src/Sylius/Bundle/ApiBundle/Command/Checkout/CompleteOrder.php

```diff
@@ -17,24 +17,14 @@
 
 class CompleteOrder implements OrderTokenValueAwareInterface
 {
-    /** @var string|null */
-    public $orderTokenValue;
-
-    /** @var string|null */
-    public $notes;
-
-    public function __construct(?string $notes = null)
-    {
-        $this->notes = $notes;
+    public function __construct(
+        public ?string $notes = null,
+        public ?string $orderTokenValue = null,
```

Comment:
```suggestion
        protected ?string $notes = null,
        protected string $orderTokenValue,
```
and the order of arguments could be changed 🤔